CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+<!--
+## Unreleased
+### Enhancements:
+### Bug Fixes:
+### Doc Fixes:
+-->
+
+## Unreleased
+
+### Enhancements:
+
+- PR-52 Masato Ikeda <masato.ikeda@gmail.com> Add MongoDB 6.0 and 7.0 to CI
+- PR-53 Masato Ikeda <masato.ikeda@gmail.com> Add Ruby 3.2 to CI matrix
```

Comment:
I changed the title from the original one "Support Ruby 3.2" to avoid misleading.
https://github.com/mongomapper/plucky/pull/53